mlir/test/Conversion/VectorToLLVM/vector-reduction-to-llvm.mlir

```diff
@@ -79,6 +79,25 @@ func.func @masked_reduce_add_f32(%arg0: vector<16xf32>, %mask : vector<16xi1>) -
 // CHECK:           "llvm.intr.vp.reduce.fadd"(%[[NEUTRAL]], %[[INPUT]], %[[MASK]], %[[VL]]) : (f32, vector<16xf32>, vector<16xi1>, i32) -> f32
 
 
+// -----
+
+func.func @masked_reduce_add_f32_scalable(%arg0: vector<[4]xf32>, %mask : vector<[4]xi1>) -> f32 {
```

Comment:
Please use identical shapes to what's used in `@masked_reduce_add_f32`. This way the only thing that changes is "scalability" rather than two things at a time. Same comment for other tests.

Reply:
Thanks, I just updated the tests.